src/api/customer/customer-route.ts

```diff
@@ -1,8 +1,10 @@
 import { Router } from 'express';
-import { createCustomer } from './customer-controller';
+import { createCustomer, findAllCustomer, findByIdCustomer } from './customer-controller';
 
 const route = Router();
 
 route.post('/v1/customers', ...createCustomer());
+route.get('/v1/customers', ...findAllCustomer());
+route.get('/v1/customers/{customerUuid}', ...findByIdCustomer());
```

Comment:
Para identificar um path parameter, o correto é usar o símbolo `:`.

```suggestion
route.get('/v1/customers/:customerUuid', ...findByIdCustomer());
```

O precise de uma documentação, express internamente usa esse padrão: https://www.npmjs.com/package/path-to-regexp